webapp/src/main/resources/public/js/components/workbench/GitBlameInfoModal.js

```diff
@@ -0,0 +1,255 @@
+            </div>
+        )
+    },
+
+    /**
+     * @returns {*} Generates the content for the debug section (additional text unit information)
+     */
+    renderDebugInfo() {
+        if (this.props.textUnit === null)
+            return "";
+        return (
+            <div className="panel-body">
+                {this.displayInfo("TmTextUnitId", this.props.textUnit.getTmTextUnitId())}
+                {this.displayInfo("TmTextUnitVariantId", this.props.textUnit.getTmTextUnitVariantId())}
+                {this.displayInfo("TmTextUnitCurrentVariantId", this.props.textUnit.getTmTextUnitCurrentVariantId())}
+                {this.displayInfo("AssetTextUnitId", this.props.textUnit.getAssetId())}
+                {this.displayInfo("LastSuccessfulAsset\nExtractionId", this.props.textUnit.getLastSuccessfulAssetExtractionId())}
+                {this.displayInfo("AssetExtractionId", this.props.textUnit.getAssetExtractionId())}
+            </div>
+        );
+    },
+
+    /**
+     * @param property The property of the GitBlame object to retrieve
+     * @returns {*}    The value of the given property of the GitBlame object
+     */
+    getGitBlameProperty(property) {
+        if (this.props.gitBlameWithUsage == null || this.props.gitBlameWithUsage["gitBlame"] == null)
+            return null;
+        return this.props.gitBlameWithUsage["gitBlame"][property];
+    },
+
+    /**
+     * @returns {str} The code author of the current text unit
+     */
+    getAuthorName() {
+        return this.getGitBlameProperty("authorName");
+    },
+
+    /**
+     * @returns {str} The email of the code author of the current text unit
+     */
+    getAuthorEmail() {
+        return this.getGitBlameProperty("authorEmail");
+    },
+
+    /**
+     * @returns {str} The commit name of the current text unit
+     */
+    getCommitName() {
+        return this.getGitBlameProperty("commitName");
+    },
+
+    /**
+     * @returns {int} The commit time (in ms) of the current text unit
+     */
+    getCommitTime() {
+        return parseInt(this.getGitBlameProperty("commitTime")) * 1000;
+    },
+
+    /**
+     * @returns {*} A list of usages of the current text unit, or null if there are none
+     */
+    getUsages() {
+        if (this.props.gitBlameWithUsage == null || this.props.gitBlameWithUsage["usages"] == null)
+            return null;
+        return this.props.gitBlameWithUsage["usages"];
+    },
+
+    /**
+     * @returns {*} Converts the list of usages of the text unit to a list of links to given text unit
+     */
+    getOpenGrokLocation() {
+        let textUnit = this.props.textUnit;
+        let usages = this.getUsages();
+
+        if (textUnit == null || usages == null)
+            return null;
+
+        if (this.props.appConfig.opengrok.server === null)
+            return this.getUsages().join('\n');
+
+        let repo = textUnit.getRepositoryName();
+
+        if (usages.length === 0) {
+            if (repo === "android")
```

Comment:
we can't have an hardcoded repository name. We need to find another solution. We can implement that in another PR